tests/bofire/strategies/test_qparego.py

```diff
@@ -142,24 +143,27 @@ def test_qparego(num_test_candidates):
 
 @pytest.mark.parametrize(
     "num_test_candidates",
-    list(range(1, 2)),
+    [1, 2],
 )
 def test_qparego_constraints(num_test_candidates):
     # generate data
-    benchmark = C2DTLZ2(dim=4)
+    benchmark = Detergent()
```

Comment:
Can you keep the test with C2DTLZ2 here? Because it is used to test if qparego also work with output constraints which are present in C2DTLZ2. You can copy the test for the input constraints and then use the `Detergent` benchmark.